_topic_maps/_topic_map.yml

```diff
@@ -202,6 +202,9 @@ Topics:
     File: uninstalling-cluster-aws
   - Name: Installation configuration parameters
     File: installation-config-parameters-aws
+  - Name: AWS Local Zone or Wavelength Zone tasks
```

Comment:
Glad to see this doc org. as these docs got a bit lost in the post-installation section. 

Optional update: Consider grabbing the reader's attention by opening the first line of the doc with "As a post-installation task.... Search engines results might benefit from this. 

I understand "Extend existing clusters to use AWS Local Zones or Wavelength Zones" states the line, but maywe it could removed from this section and applied to the opening sentence of the doc?

![Screenshot from 2024-08-28 14-34-12](https://github.com/user-attachments/assets/bcfd78bf-08f7-499e-94f5-104509b1d043)

Reply:
I'll leave this for now just because I want to avoid making any changes to the existing content where possible - the PR is really just to move the assembly and update links as needed 🙂